packages/next/src/generators/library/library.ts

```diff
@@ -33,9 +33,28 @@ export async function libraryGenerator(host: Tree, options: Schema) {
     ...options,
     compiler: 'swc',
     skipFormat: true,
+    // Always include deep import paths so users can mix server and client react components in the same library.
+    // See: https://github.com/nrwl/nx/issues/15830
+    additionalImportPaths: ['server'],
```

Comment:
I think entry points work better? It's like multiple entry points into this project.

```suggestion
    additionalEntryPoints: ['server'],
```

I kind of see why it's in the `react:library` since I guess it enables RSC... why not just do everything in here for now though?

I don't like how a blank file is created in the react library.. and then this has to write the same exact file.

Reply:
I think it's too hard to know what should be in the file since it's very Next.js specific. The React generator can only generate blank files as to not assume what's in there. When RSC is more stable there might be some consensus or standard on how libraries should be built.

Reply:
Updated it so this option is not added anymore. Instead, the Next.js library generator will add the file + import path that is specific to Next.js apps.